### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dajoun1**
# Definition of Done

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,50 +3,74 @@
...
+    *  System Enviroment Variable JavaWarriorsUser = Database username
+    *  System Enviroment Variable JavaWarriors = Database password
+      
+# Setting up the application's properties
+    Inside application.properties, paste this:
+
+    server.port=9003
+
+    spring.datasource.url = jdbc:postgresql://localhost:5432/LayoutTeam
+    spring.datasource.username = ${Database username environment variable}
```
Comment (JoneyKroney) [inline README.md:65]: change this

**Review by JoneyKroney — CHANGES_REQUESTED** [review]

**Review by Dajoun1 — COMMENTED** [review]
See if its fixed @JoneyKroney @Ethanm46

File: README.md (hunk 1 above)
Comment (Dajoun1) [reply README.md]: Done

**Review by JoneyKroney — APPROVED** [review]
:)

**Pull request merged**